src/NHibernate/Loader/Hql/QueryLoader.cs

```diff
@@ -119,7 +119,7 @@ protected override bool[] EntityEagerPropertyFetches
 			get { return _entityEagerPropertyFetches; }
 		}
 
-		protected override HashSet<string>[] EntityFetchLazyProperties
+		protected override ISet<string>[] EntityFetchLazyProperties
```

Comment:
Is this released yet?

Reply:
No. All public changes from unreleased #1922